build.ps1

```diff
@@ -0,0 +1,80 @@
+param (
+    [Parameter(Mandatory=$true)][string]$Project,
+    [Parameter(Mandatory=$true)][string]$Package,
+    [string]$MinimumGameVersion = "1.7.14",
+    [string]$OutputPath = ".\build\"
+)
+
+[XML]$projectFile = Get-Content $Project
+$packages = @{}
+
+foreach($packageEntry in $projectFile.Project.Packages.Package) {
+    [XML]$packageFile = Get-Content $packageEntry
+    $packages[$packageFile.AssetPackage.Name] = $packageFile
+}
+
+if ($packages.ContainsKey($Package)) {
+    $packageDef = $packages[$Package].AssetPackage
+
+    $manifest = New-Object -TypeName PSObject -Property @{
+        dependencies = @()
```

Comment:
Just flagging for future work that we talked about being able to include dependencies for our own packages, this'll need to be enhanced for that at some point.